src/common/lock.c

```diff
@@ -284,3 +284,32 @@ lockRelease(bool failOnNoLock)
 
     FUNCTION_LOG_RETURN(BOOL, result);
 }
+
+/***********************************************************************************************************************************
+Look for a backup lock locally, for a specific stanza
+***********************************************************************************************************************************/
+
+String *
+lockTypeBackupFound(const String *lockPath, const String *stanza)
+{
+    FUNCTION_LOG_BEGIN(logLevelDebug);
+        FUNCTION_LOG_PARAM(STRING, lockPath);
+        FUNCTION_LOG_PARAM(STRING, stanza);
+    FUNCTION_LOG_END();
+
+
+    // Lock file to found
+    String *lockFileToFind = strNewFmt(
+        "%s/%s-%s" LOCK_FILE_EXT, strPtr(lockPath), strPtr(stanza), lockTypeName[lockTypeBackup]);
+
+    // Try to open the lock file
+    String *backupProcessId = NULL;
+    IoRead *source = storageReadIo(storageNewReadP(storageLocal(), lockFileToFind, .ignoreMissing = 1));
+    if (ioReadOpen(source))
+    {
+        backupProcessId = ioReadLine(source);
+        ioReadClose(source);
+    }
+
+    FUNCTION_LOG_RETURN(STRING, backupProcessId);
+}
```

Comment:
Since we are not going to capture the pid I think we remove this and just use `lockAcquire()` and `lockRelease()`.